app/models/skin.rb

```diff
@@ -442,6 +459,7 @@ def self.load_site_css
         File.open(version_dir + 'preview.png', 'rb') {|preview_file| top_skin.icon = preview_file}
         top_skin.official = true
         top_skin.save!
+        Rails.cache.increment('skins_generation/'+top_skin.type.nil? ? ("site_skin") : top_skin.id.to_s )
```

Comment:
Surrounding space missing for operator `+`.<br>Space inside parentheses detected.